anomalib/models/fastflow/README.md

```diff
@@ -0,0 +1,124 @@
+# FastFlow: Unsupervised Anomaly Detection and Localization via 2D Normalizing Flows
+
+This is the implementation of the [FastFlow](https://arxiv.org/abs/2111.07677) paper. This code is developed inspired by [https://github.com/gathierry/FastFlow](https://github.com/gathierry/FastFlow).
+
+Model Type: Segmentation
+
+## Description
+
+FastFlow is a two-dimensional normalizing flow-based probability distribution estimator. It can be used as a plug-in module with any deep feature extractor, such as ResNet and vision transformer, for unsupervised anomaly detection and localisation. In the training phase, FastFlow learns to transform the input visual feature into a tractable distribution, and in the inference phase, it assesses the likelihood of identifying anomalies.
+
+## Architecture
+
+![FastFlow Architecture](../../../docs/source/images/fastflow/architecture.jpg "FastFlow Architecture")
+
+## Usage
+
+`python tools/train.py --model fastflow`
+
+## Benchmark
+
+All results gathered with seed `0`.
+
+## [MVTec AD Dataset](https://www.mvtec.com/company/research/datasets/mvtec-ad)
+
+---
+**NOTE**
+
+When the numbers are produced, early stopping callback (patience: 3) is used. It might be possible to achieve higher-metrics by increasing the patience.
+
+---
+
+### Image-Level AUC
+
+|            | ResNet-18 | Wide ResNet50 | DeiT  | CaiT  |
+| ---------- | :-------: | :-----------: | :---: | :---: |
+| Bottle     |   1.000   |     1.000     | 0.905 | 0.986 |
+| Cable      |   0.891   |     0.962     | 0.942 | 0.839 |
+| Capsule    |   0.900   |     0.963     | 0.819 | 0.913 |
+| Carpet     |   0.979   |     0.994     | 0.999 | 1.000 |
+| Grid       |   0.988   |     1.000     | 0.991 | 0.979 |
+| Hazelnut   |   0.846   |     0.994     | 0.900 | 0.948 |
+| Leather    |   1.000   |     0.999     | 0.999 | 0.991 |
+| Metal_nut  |   0.963   |     0.995     | 0.911 | 0.963 |
+| Pill       |   0.916   |     0.942     | 0.910 | 0.916 |
+| Screw      |   0.521   |     0.839     | 0.705 | 0.791 |
+| Tile       |   0.967   |     1.000     | 0.993 | 0.998 |
+| Toothbrush |   0.844   |     0.836     | 0.850 | 0.886 |
+| Transistor |   0.938   |     0.979     | 0.993 | 0.983 |
+| Wood       |   0.978   |     0.992     | 0.979 | 0.989 |
+| Zipper     |   0.878   |     0.951     | 0.981 | 0.977 |
+| Average    |           |               |       |       |
```

Comment:
Can you add the average score? Also, a minor issue but this format is a bit different from our other models (rows <-> columns). Maybe for consistency we should decide on a single format. Maybe this can be addressed in a different PR.

Reply:
We could perhaps modify all the tables once we have all the benchmark results merged